build.xml

```diff
@@ -535,7 +539,11 @@ Authors:
       doctitle='Xerces Native Interface'
       bottom='${copyright}'
       additionalparam='-quiet ${additional.param}'
-      />
+      failonerror='yes'
+      failonwarning='yes'
```

Comment:
No. I do not want the build to fail on a warning. I'm not sure it should fail on an error here, but definitely not on a warning

Reply:
I've only set yes on the sections that don't have any errors or warnings. This branch passes, so setting yes here is to ensure future changes don't introduce problems.

Reply:
Saying that, I just saw that behaviour is different on the CI build. I'll take a look at why it's passing locally but not on CI